src/js/literal-map.js

```diff
@@ -0,0 +1,55 @@
+const { get, getOwnPropertyDescriptor, ownKeys } = Reflect;
+
+const getPropertyDescriptor = (value) => ({
+  value,
+  enumerable: true,
+  writable: true,
+  configurable: true,
+});
+
+const _ = Symbol();
+const prototype = "prototype";
+
+const handler = {
+  deleteProperty: (map, k) => (map.has(k) ? map.delete(k) : delete map[k]),
+  get(map, k, proxy) {
+    if (k === _) return map;
+    let v = map[k];
+    if (typeof v === "function" && k !== "constructor") {
```

Comment:
this is shadowing explicit `get` and `set` methods on the literal ... there are not many use cases for these or known APIs but properties descriptors have both `get` and `set` possible.

I have [explained in the README](https://github.com/WebReflection/literal-map?tab=readme-ov-file#differently-from-python-dictionaries) why the module instead prefers own properties over inherited methods, also because that's how it works in general in JS via prototypal inheritance.

a `{get: 123}` will also fully break here expectations.

Please re-consider this change as I am not sure it should land in my module too.

Reply:
FYI MicroPython is also considering to switch to this module to make the Python/JS story more portable across interpreters https://github.com/micropython/micropython/pull/13583#issuecomment-1972480047 **but** if this interpreter decides that `{"get": py_method}` should break, when explicit, in favor of the underlying `Map.prototype.get` I think my proposal would fail if not used *as-is* and prefer other interpreters to just convert by default as object literals, differently from Pyodide ... let's please try to find out the best of both worlds we can offer, considering the current de-facto standard in JS APIs is that config options and parameters are expected to be object literals and that a destructured `get` in a *Map* would hardly fail no matter what if expected as value or non-context dependent method due new or foreign JS API design, thank you!